plugins/elasticsearch/loggable-transport-netty4/build.gradle

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+wrapper {
+   distributionType = Wrapper.DistributionType.ALL
+}
+
+buildscript {
+  ext {
+    version_es = "7.10.2"
+  }
+
+  repositories {
+    mavenCentral()
+    jcenter()
+  }
+
+  dependencies {
+    classpath "org.elasticsearch.gradle:build-tools:${version_es}" 
+  }
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+  }
+
+apply plugin: "elasticsearch.esplugin"
+
+dependencies {
+  implementation "org.elasticsearch.plugin:transport-netty4-client:${version_es}"
+}
+
+esplugin {
+  name "LoggableNetty4-${version_es}"
+  description "Plugin to override the transport-netty4 module surgically to add logging of all HTTP requests"
+  classname "org.opensearch.transport.LoggableNetty4ModulePlugin"
+  licenseFile rootProject.file("licenses/APACHE-LICENSE-2.0.txt")
+  noticeFile rootProject.file("NOTICE.txt")
+}
```

Comment:
This file looks very different than the `build.gradle` in the "matching" opensearch path: https://github.com/opensearch-project/opensearch-migrations/pull/114/files#diff-161d6851fc48752fb3be01e166eccf15ad8a2eef55e8c113fd47e9c76dc22a65R30. I don't know Gradle at all -- can you explain why they're different (are they doing different things?) and whether they rely on each other?

Reply:
They're mostly different because of the way ES plugins are built compared to OS plugins, even "importing" certain dependencies are different within Gradle.
Each of the Gradle files within this commit are standalone and don't rely on each other.